File: app/src/main/java/org/michaelbel/application/rest/model/Review.java
package org.michaelbel.application.rest.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

@SuppressWarnings("all")
public class Review implements Serializable {

    @SerializedName("id")
    public String id;

    @SerializedName("author")
    public String author;

    @SerializedName("content")
    public String content;

    @SerializedName("iso_639_1")
    public String lang;

    @SerializedName("media_id")
    public int mediaId;

    @SerializedName("media_title")
    public String mediaTitle;

    @SerializedName("media_type")
    public String mediaType;

    @SerializedName("url")
    public String url;
}

File: app/src/main/java/org/michaelbel/moviemade/data/entity/Review.kt
package org.michaelbel.moviemade.data.entity

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Review(
    @SerializedName("id") val id: String,
    @SerializedName("author") val author: String,
    @SerializedName("content") val content: String,
    @SerializedName("iso_639_1") val lang: String,
    @SerializedName("media_id") val mediaId: Int,
    @SerializedName("media_title") val mediaTitle: String,
    @SerializedName("media_type") val mediaType: String,
    @SerializedName("url") val url: String
) : Serializable